python/python/lancedb/table.py

```diff
@@ -2252,20 +2252,28 @@ async def update(
         0  1  [1.0, 2.0]
         1  2  [3.0, 4.0]
         2  3  [5.0, 6.0]
-        >>> table.update(where="x = 2", values={"vector": [10, 10]})
+        >>> table.update({"vector": [10, 10]}, where="x = 2")
```

Comment:
Looks like this is failing in doctest because table here is still the sync version with the old signature.

Reply:
Ah, thanks.  Fixed.  Sadly, it doesn't seem there is any great way to use doctest with asyncio.  Best I could find is...

```
>>> import asyncio
>>> async def some_fn():
...     await do_async_stuff()
>>> asyncio.run(some_fun())
```